File: Source/prx/Source/Library/PrxSample/prx/src/main/java/com/philips/cdp/prxclient/datamodels/features/KeyBenefitAreaItem.java
package com.philips.cdp.prxclient.datamodels.features;

import java.io.Serializable;
import java.util.List;
import com.google.gson.annotations.SerializedName;

public class KeyBenefitAreaItem implements Serializable {

	private static final long serialVersionUID = -7309764568406054211L;
	@SerializedName("feature")
	public List<FeatureItem> feature;

	@SerializedName("keyBenefitAreaCode")
	public String keyBenefitAreaCode;

	@SerializedName("keyBenefitAreaName")
	public String keyBenefitAreaName;

	@SerializedName("keyBenefitAreaRank")
	public String keyBenefitAreaRank;
}

File: Source/prx/Source/Library/PrxSample/prx/src/main/java/com/philips/cdp/prxclient/datamodels/features/KeyBenefitAreaItem.kt
package com.philips.cdp.prxclient.datamodels.features

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class KeyBenefitAreaItem(var feature: List<FeatureItem>? = null, var keyBenefitAreaCode: String? = null, var keyBenefitAreaName: String? = null, var keyBenefitAreaRank: String? = null) : Parcelable